.expeditor/config.yml

```diff
@@ -0,0 +1,15 @@
+# Documentation available at https://expeditor-docs.es.chef.io/
+
+github:
+  # The file where the MAJOR.MINOR.PATCH version is kept. The version in this file
+  # is bumped automatically via the `built_in:bump_version` merge_action.
+  version_file: "VERSION"
+  # The file where our CHANGELOG is kept. This file is updated automatically with
+  # details from the Pull Request via the `built_in:update_changelog` merge_action.
+  changelog_file: "CHANGELOG.md"
+  # The Github Team primarily responsible for handling incoming Pull Requests.
+  maintainer_group: habitat-sh/habitat-core-maintainers
+
+# These actions are taken, in order they are specified, anytime a Pull Request is merged.
+merge_actions:
+  - built_in:update_changelog:
```

Comment:
You'll probably want to add an [`ignore_labels`](http://expeditor-docs.es.chef.io/configuration/merge-actions/#ignore_labels) section, just in case you're merging a PR that you do not want in your changelog.

Reply:
I don't think we have anything like that now, but presumably we can add later if needed, yes?